src/enum_derive.rs

```diff
@@ -279,10 +290,19 @@ fn expand(args: TokenStream, input: TokenStream) -> Result<TokenStream> {
 
     let mut derive = vec![];
     let mut items = TokenStream::new();
-    let cx = DeriveContext::default();
+    let mut cx = DeriveContext::default();
     for (s, arg) in args {
+        #[cfg(not(feature = "convert"))]
+        if s == "Into" {
+            return Err(Error::new(
+                proc_macro2::Span::call_site(),
+                "`Into` derive is only supported via `convert` feature.",
+            ));
+        }
```

Comment:
I'm positive about generating such errors, but rather than hard-coding something for a specific trait here, I would like to adopt an approach that also works for other feature-gated traits.